vertx-mysql-client/src/test/java/io/vertx/mysqlclient/MySQLQueryTest.java

```diff
@@ -38,6 +39,133 @@ public void teardown(TestContext ctx) {
     vertx.close(ctx.asyncAssertSuccess());
   }
 
+  @Test
+  public void testTableCollation(TestContext ctx) {
+    MySQLConnection.connect(vertx, options, ctx.asyncAssertSuccess(conn -> {
+      conn.query("CREATE TEMPORARY TABLE chinese_city (\n" +
+        "\tid INTEGER,\n" +
+        "\tcity_name VARCHAR(20)) COLLATE gbk_chinese_ci;\n" +
```

Comment:
we need to use unicode escape for the Chinese chars,  i.e `\uXXX` so they work with any file editor

Reply:
Fixed in https://github.com/eclipse-vertx/vertx-sql-client/pull/352/commits/1f91501eb1ccb871e05f41adef1cb6b59f094aec.